src/components/NcRichText/helpers.js

```diff
@@ -1,4 +1,4 @@
-export const URL_PATTERN = /(\s|^)(https?:\/\/)((?:[-A-Z0-9+_]+\.)+[-A-Z]+(?:\/[-A-Z0-9+&@#%?=~_|!:,.;()]*)*)(\s|$)/ig
+export const URL_PATTERN = /(\s|^)(https?:\/\/)((?:[-A-Z0-9+_]+\.)+[-A-Z0-9]+(?:\/[-A-Z0-9+&@#%?=~_|!:,.;()]*)*)(\s|$)/ig
```

Comment:
IMO this should not be changed. local urls should not be resolved, anyway the server reference manager refuses to resolve.

Reply:
This is not about the reference manager. it's the "make clickable".
Also posting a fully public IP is totally fine from my POV.